### PR TITLE
milestone three: Kelwilson-portfolio

### PR DESCRIPTION
**Pull request by kelwilson**
###  **form-field ;**

- Add a section with the class form section
- Add the `<h1>`tag and `<p>`tag with some content about how to get started
- Add a form field with a text, email, and message field
- Add a submit `<button>` to the form field
- Add a `<footer>` tag with a `<nav>` tag with a list of social media links icons
- Add a display flex to the form section class to make it a container, and add font styles to the content of the  `<h1>` and `<p>` tag
- Add width padding margin and some font styles to the placeholder texts in the form field
- Add some styles to the nav links some padding and color to the link list
- snapshot of the for-field
form

### CODE REVIEW

**Comment by kelwilson** [comment]
good day, please I tried removing the white lines beside the icons I imported from font awesome but didn't find a way to do it, please I would be glad if you could help me with a hint on how to go about it. here is a screenshot below of what I'm actually talking about, the little lines beside the angelic icon the medium icon and the StackOverflow icon
form 2
thanks for your time and I'm always open to constructive criticism

File: style.css [hunk 1]
```diff
@@ -446,3 +447,177 @@
 .bx-download:disabled {
   background: #1a2236;
 }
+
+#form-section {
```
Comment (arslanbisharat) [inline style.css]: - [ ] It's better to use classes instead of IDs in your CSS.

File: style.css [hunk 2]
```diff
@@ -446,3 +447,177 @@
...
+}
+
+.portfolio-form {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 20px;
+}
+
+#Username {
```
Comment (arslanbisharat) [inline style.css]: - [ ] It's better to use classes instead of IDs in your CSS.

File: style.css [hunk 3]
```diff
@@ -446,3 +447,177 @@
...
+  border-radius: 4px;
+  margin-bottom: 8px;
+  font-size: 13px;
+  font-family: "Poppins", sans-serif;
+  padding-left: 16px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+}
+
+#email {
```
Comment (arslanbisharat) [inline style.css]: - [ ] It's better to use classes instead of IDs in your CSS.

File: style.css [hunk 4]
```diff
@@ -446,3 +447,177 @@
...
+  border-radius: 4px;
+  margin-bottom: 8px;
+  font-size: 13px;
+  font-family: "Poppins", sans-serif;
+  padding-left: 16px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+}
+
+#message {
```
Comment (arslanbisharat) [inline style.css]: - [ ] It's better to use classes instead of IDs in your CSS.

File: style.css [hunk 5]
```diff
@@ -446,3 +447,177 @@
...
+.footer-nav {
+  margin-top: 70px;
+  background: #1a2236;
+  display: flex;
+  flex-direction: column;
+  height: 250px;
+  justify-content: center;
+}
+
+#resume {
```
Comment (arslanbisharat) [inline style.css]: - [ ]  It's better to use classes instead of IDs in your CSS.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @kelwilson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
> good day, please I tried removing the white lines beside the icons I imported from font awesome but didn't find a way to do it, please I would be glad if you could help me with a hint on how to go about it. here is a screenshot below of what I'm actually talking about, the little lines beside the angelic icon the medium icon and the StackOverflow icon form 2 thanks for your time and I'm always open to constructive criticism

Thank you for the question. I am looking into it. I'll get back to you as soon as possible.  @TedLivist

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @kelwilson,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**